deployment/src/main/java/com/tietoevry/quarkus/resteasy/problem/deployment/ProblemProcessor.java

```diff
@@ -25,62 +25,59 @@
     private static final String FEATURE_NAME = "resteasy-problem";
     private static final String PACKAGE = "com.tietoevry.quarkus.resteasy.problem.";
 
-    private static final Map<String, String> DEFAULT_MAPPERS = defaultMappers();
+    private static final Logger logger = Logger.getLogger(FEATURE_NAME);
 
-    private static Map<String, String> defaultMappers() {
+    private static Map<String, String> neededExceptionMappers() {
         Map<String, String> mappers = new LinkedHashMap<>();
         mappers.put("java.lang.Exception", "DefaultExceptionMapper");
-        mappers.put(PACKAGE + "HttpProblem", "HttpProblemMapper");
+        mappers.put("com.tietoevry.quarkus.resteasy.problem.HttpProblem", "HttpProblemMapper");
         mappers.put("javax.ws.rs.WebApplicationException", "jaxrs.WebApplicationExceptionMapper");
         mappers.put("javax.ws.rs.ForbiddenException", "jaxrs.JaxRsForbiddenExceptionMapper");
         mappers.put("javax.ws.rs.NotFoundException", "jaxrs.NotFoundExceptionMapper");
         mappers.put("io.quarkus.security.UnauthorizedException", "security.UnauthorizedExceptionMapper");
         mappers.put("io.quarkus.security.AuthenticationFailedException", "security.AuthenticationFailedExceptionMapper");
         mappers.put("io.quarkus.security.ForbiddenException", "security.ForbiddenExceptionMapper");
-        return Collections.unmodifiableMap(mappers);
-    }
-
-    private static final Map<String, String> VALIDATION_MAPPERS = validationMappers();
-
-    private static Map<String, String> validationMappers() {
-        Map<String, String> mappers = new LinkedHashMap<>();
         mappers.put("javax.validation.ValidationException", "javax.ValidationExceptionMapper");
         mappers.put("javax.validation.ConstraintViolationException", "javax.ConstraintViolationExceptionMapper");
-        return Collections.unmodifiableMap(mappers);
+        mappers.put("org.zalando.problem.ThrowableProblem", "ZalandoProblemMapper");
+
+        mappers.put("com.fasterxml.jackson.core.JsonProcessingException", "jackson.JsonProcessingExceptionMapper");
+        mappers.put("javax.ws.rs.ProcessingException", "jsonb.RestEasyClassicJsonbExceptionMapper");
+        mappers.put("javax.json.bind.JsonbException", "jsonb.JsonbExceptionMapper");
+
+        return mappers.entrySet().stream()
+                .filter(mapper -> {
+                    String exceptionClass = mapper.getKey();
+                    return classAvailable(exceptionClass);
+                })
```

Comment:
Maybe it's better to extract whole logic of lambda and put it to one method called `isClassAvailable()` or `hasRegisteredClass()` or `isOnClasspath()`
For me it's always better to have a prefix for all boolean methods i.e `is` or `has` ...

Reply:
I can add `is`, no problem. Remember though, that this method accepts either exception class name, or not-very-readable `Map.Entry`, as we're filtering map, not a list. 

Thus:`filter` accepts `Predicate<Map.Entry>`, not `Predicate<String>`. I find it more readable extracting entry.getKey into properly named variable first (`exceptionClass`), then passing it to final method.